tests.toml

```
@@ -13,7 +13,10 @@ test_format = 1.0
 # (Remove this key entirely if not needed)
 preinstall = """
     sudo yunohost tools update apps
-    sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh -a "domain=$domain&init_main_permission=all_users&server_name=$server_name&is_free_registration=$is_free_registration&jitsi_server=$jitsi_server" --force
+    sudo yunohost domain add matrix.domain.tld
+    domain=matrix.domain.tld
+    server_name=$domain
+    sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh -a "domain=$domain&init_main_permission=all_users&server_name=$server_name&is_free_registrationfalse" --force
```

```suggestion
    sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh -a "domain=$domain&init_main_permission=all_users&server_name=$server_name&is_free_registration=false" --force
```

---

I could achieve to make this work on my package by using plain values instead of variables:
https://github.com/YunoHost-Apps/mautrix_discord_ynh/pull/27/files

```suggestion
    sudo yunohost app install https://github.com/YunoHost-Apps/synapse_ynh -a "domain=domain.tld&init_main_permission=all_users&server_name=domain.tld&is_free_registration=false" --force
```

---

thanks